components/docs-chef-io/content/automate/chef_infra_server.md

```diff
@@ -89,13 +89,108 @@ A [cookbook]({{< relref "cookbooks" >}}) is the fundamental unit of configuratio
 
 An [environment]({{< relref "environments" >}}) can be used to map an organization's real-life workflow to what can be configured and managed when using Chef Infra. This mapping is accomplished by setting attributes and pinning cookbooks at the environment level.
 
+The Chef Infra Server UI lets you:
+
+- Create an environment.
+- Search for a specific environment from a list of environments
+- List out all the environments at one place.
+- View the details of an environment.
+- Edit an environment.
+- Delete an environment.
+
+#### Create an Environment
+
+To create a new Environment, select **Create Environment**, as shown below:
+
+{{< figure src="/images/automate/create-environment-button.png" alt="Create Environment Button">}}
+
+Selecting the **Create Environment** button opens a dialog box. The dialog box contains four different sections named _Details_, _Constraints_, _Default Attributes_ and _Override Attributes_. Enter the details in the possible sections and select **Create** to create a new environment.
+
+{{< figure src="/images/automate/create-environment-popup.png" alt="Create Environment Dialog Box">}}
+
+#### Search for an Environment
+
+Use the **Search environments** bar to find an existing environment from the list of environments.
+Entering the name of an environment in the search box returns environments matching your search criteria.
+
+{{< figure src="/images/automate/create-environment-button.png" alt="Create Environment Button">}}
+
+#### Details of an Environment
+
+To view the details of an environment, follow the steps mentioned below:
+
+- Select a specific environment.
+  - **Cookbook Constraints:** The selected bar contains the list of all the cookbook constraints with _Name_, their _Operator_, and _Version_ respectively. Select **Edit** to change the _Cookbook Constraints_.
```

Comment:
This is a bit vague. I assume it is listing each cookbook in an environment by name, followed by the cookbook version number and the version constraint. We might want to link to https://docs.chef.io/cookbook_versioning/#constraints

```suggestion
**Cookbook Constraints**

The Cookbook Constraints window lists each cookbook by name followed by it's [version constraint]({{< relref "cookbook_versioning#constraints">}}) and version number.
```